x-pack/test/apm_api_integration/tests/index.ts

```diff
@@ -120,11 +120,15 @@ export default function apmApiIntegrationTests(providerContext: FtrProviderConte
       loadTestFile(require.resolve('./services/error_groups/error_groups_detailed_statistics'));
     });
 
+    describe('sservices/errors/error_group_list', function () {
+      loadTestFile(require.resolve('./services/errors/error_group_list'));
```

Comment:
The folder "./services/error_groups" already exists. Maybe it should go there instead of "./services/errors" ?

Reply:
I think it makes more sense if I moved it to the errors folder out of the services folder,  @cauemarcondes created it for the api test that are testing the endpoint with `[serviceName]/errors/...`